### DISCUSSION TITLE
pydeck how to use on_view_state_change with orthographic view

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5790] [Q&A] [answered]
Can we use [on_view_state_change](https://pydeck.gl/event_handling.html#view-state-change-events-with-on-view-state-change) to trigger events when using an orthographic view?

```
view = pdk.View(
    type="OrthographicView",
    controller=True, 
)
```

If so, would we still have access to the bounds like 'nw', 'se', etc?

**Comment by cornhundred** [comment, marked as answer]
I followed the instructions to inspect the payload using an HTML widget using this function 

```
def filter_by_viewport(widget_instance, payload):
    text.value = str(payload)
```

and saw outputs like this

```
{'type': 'deck-view-state-change-event', 'data': {'width': 985, 'height': 500, 'orbitAxis': 'Z', 'rotationX': 0, 'rotationOrbit': 0, 'target': [363.30811230604013, 362.781105203365, 4.490630090003833e-12], 'zoom': 0.24283613435629614, 'minRotationX': -90, 'maxRotationX': 90, 'minZoom': None, 'maxZoom': None, 'transitionDuration': 1, 'transitionInterpolator': {'_propsToCompare': ['target', 'zoom'], '_propsToExtract': ['target', 'zoom'], '_requiredProps': ['target', 'zoom'], 'opts': ['target', 'zoom']}, 'transitionInterruption': 1, 'nw': [-292.6427802433585, 81.44229149247451], 'se': [292.6427802433591, -81.44229149247454]}}
```

So it looks like I have the 'nw' and 'se' bounds.

**Reply by cornhundred** [reply]
Although the 'nw' and 'se' bounds appear to be opposite to each other and only change when zooming and not panning. I'm using pydeck version 0.6.2

**Reply by ajduberstein** [reply]
Thanks for reporting @cornhundred. I'll investigate.
